recipes/jupyterhub/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.6.1" %}
+
+package:
+  name: jupyterhub
+  version: {{ version }}
+
+source:
+  fn: jupyterhub-{{ version }}.tar.gz
+  url: https://pypi.python.org/packages/ac/9b/ef3435cf8951d7463f4175a0066a8454e224a598b2159f521cc011f8c2b6/jupyterhub-{{ version }}.tar.gz
+  md5: 6bd795a82bf15a037d097991110b8713
+
+build:
+  number: 0
+  skip: true  # [win or py2k]
+  script: pip install --no-deps .
```

Comment:
@jakirkham we already have `pip install --no-deps .` in a few feedstocks and it is virtually the same as `python setup.py install --single-version-externally-managed --record=record.txt`. I am inclined to leave it as is and start the move to `pip` slowly.